code/modules/surgery/organs/tongue.dm

```diff
@@ -96,7 +97,7 @@
 	say_mod = "hisses"
 	taste_sensitivity = 10 // combined nose + tongue, extra sensitive
 	modifies_speech = TRUE
-	languages_native = list(/datum/language/draconic)
+	languages_native = list(/datum/language/draconic, /datum/language/ashtongue) //SKYRAT EDIT: Ashtongue for Ashwalkers
```

Comment:
doesn't this give regular lizardpeople ashtongue?

Reply:
No, this just makes it so that they don't hiss.